Lib/glyphsLib/parser.py

```diff
@@ -33,14 +33,32 @@ def __init__(self, current_type=OrderedDict, format_version=2):
     def parse(self, d):
         try:
             if isinstance(d, str):
+                d = self._fl7_format_clean(d)
                 d = openstep_plist.loads(d, use_numbers=True)
             elif isinstance(d, bytes):
+                d = self._fl7_format_clean(d)
                 d = openstep_plist.loads(d.decode(), use_numbers=True)
             result = self._parse(d)
         except openstep_plist.parser.ParseError as e:
             raise ValueError("Failed to parse file") from e
         return result
 
+    def _fl7_format_clean(self, d):
+        """FontLab 7 glyphs source format exports include a final closing semicolon.
+        This method removes the semicolon before passing the string to the parser."""
+        # see https://github.com/googlefonts/fontmake/issues/806
+        if isinstance(d, str):
+            if d.endswith(";"):
+                d = d.rstrip(";")
+            elif d.endswith(";\n"):
+                d = d.rstrip(";\n")
```

Comment:
```suggestion
            d = d.rstrip(";\n")
```

With one call to rstrip it would handle all the cases

Reply:
Ty!

https://github.com/chrissimpkins/glyphsLib/commit/56a428942ccbd4ae9b227dd69a779883ac6afa5d